components/BottomNavigation/examples/BottomNavigationResetButtons.swift

```diff
@@ -0,0 +1,129 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import Foundation
+import MaterialComponents
+
+class BottomNavigationResetExample: UIViewController {
+
+  var colorScheme = MDCSemanticColorScheme()
+
+  let bottomNavBar = MDCBottomNavigationBar()
+
+  // The tabs that will be reordered later
+  let tabBarItem1 = UITabBarItem(title: "Home", image: UIImage(named: "Home"), tag: 0)
+  let tabBarItem2 =
+    UITabBarItem(title: "Messages", image: UIImage(named: "Email"), tag: 1)
+  let tabBarItem3 =
+    UITabBarItem(title: "Favorites", image: UIImage(named: "Favorite"), tag: 2)
+  let tabBarItem4 = UITabBarItem(title: "Cake", image: UIImage(named: "Cake"), tag: 3)
+
+  let buttonOne = MDCButton()
+  let buttonTwo = MDCButton()
+
+  init() {
```

Comment:
Is this required for Swift? Isn't the superclass default behavior of `-init` to call `-initWithNibName:bundle:`?

Reply:
This is needed or a crash will occur.

Reply:
I don't get a crash when I try:

```objectivec
id bar = [[MDCBottomNavigationBar alloc] init];
```

Is this a bug in CatalogByConvention?

Reply:
I think it's only an issue with Swift

Reply:
Ack.